doc/SAPV.md

```diff
@@ -0,0 +1,138 @@
+In order for the source host to also validate the path, the
+destination host sends back a response packet containing the updated
+HVFs. Path validation for the source host is optional however, and
+needs to be explicitly enabled.
+
+The computation of the HVFs and the SPAO authenticator is based on DRKey.
+
+## Procedures
+
+### Control Plane
+
+In the control plane, the ASes do not only append 6 bytes of the hop
+authenticators to the beacon, but also the remaining 10 bytes (the
+authenticator is the 16 byte long output of a MAC function).
+
+### Data Plane
+
+The data plane operations for EPIC-SAPV path type packets are the same
+as for SCION path type packets, but the source additionally computes
+the per-packet HVFs for every AS on the path (for
+which it needs the full 16 bytes of the hop authenticators) plus the
+SPAO.
+All on-path border routers then validate the HVFs accordingly. In
+addition, every egress border router updates (rewrites) its
+corresponding hop validation field with a proof that it processed the
+packet. The destination host checks the SPAO, which allows it to
+verify the authenticity of the packet (because source and payload are
+authenticated), and to validate the path (because also the updated HVFs
+are authenticated). If desired by the source, the destination host
+sends back a response packet containing the updated HVFs, which allows
+the source to also verify the path.
+A more concise description can be found in the EPIC-SAPV path
+type specification.
+
+### Response Packets
+
+Similar to SCION and EPIC-HP path type packets, also EPIC-SAPV
+allows to invert the path to enable immediate communication to the
+source host. Such reversed EPIC-SAPV packets will however only be
+treated as best-effort. Alternatively, an AS can also simply fetch a
+new SCION or EPIC-SAPV path for the traffic in the backward direction.
+
+### Registering Path Segments
+
+An AS can register its path segment for the use of EPIC-SAPV, or as
+SCION-only. In the first case, the AS registers the segment including
+the full 16 bytes of the hop authenticators. In the second case, it
+only keeps the first 6 bytes of the hop authenticator, i.e., it
+removes the 10 additional bytes. Note that when all hop authenticators
+are truncated except for the last two, EPIC-SAPV communication is
+not possible, but EPIC-HP still is.
+
+As with EPIC-HP, also EPIC-SAPV traffic can be used for hidden-path
+communication [[2]](#2): an AS can distribute the authenticators to
+a set of trusted sources only, that will be the only ones able
+to send EPIC-SAPV traffic over the segment.
+
+Note that an AS cannot publicly register a segment for EPIC-SAPV
+(i.e., with the full authenticators) and at the same time distribute
+it as a hidden path for EPIC-HP.
+
+### Duplicate Suppression
+
+To protect against replay attacks, EPIC-SAPV assumes that a duplicate
+suppression system is deployed at each AS [[3]](#3).
```

Comment:
After the book is finished, a reference to the corresponding section there would be better